rebabel_format/converters/nlp_part_of_speech.py

```diff
@@ -5,41 +5,72 @@
 
 
 class NLPPartOfSpeechReader(LineReader):
-    """Read in a text file.
-    Each line consists of a single sentence made up of words tagged with their parts of speech.
-    Each word is separated from its part of speech by a delimiter, by default "/".
-    The words are separated from each other by spaces.
+    """
+    Supports two different methods for importing part of speech data.
+
+    First method:
+        Import one text file, with a "combined" nlpFileType, where each line consists of a single sentence
+        made up of words tagged with their parts of speech.
+        Each word is separated from its part of speech by a delimiter, by default "/".
+        The word/pos pairs are separated from each other by spaces.
+
+        Example file structure:
+
+        The/DET dog/NOUN barked/VERB ./PUNC
+        The/DET cat/NOUN meowed/VERB ./PUNC
+
+    Second method:
+        Import two text files, one with the original language data and one with the parts of speech.
+        The language data file (with a "language" nlpFileType) and the part of speech file
+        (with a "pos" nlpFileType) need to be uploaded in two different imports.
+        Either the language data or the part of speech file can be uploaded first, but in either case
+        the second import needs to include the merge_on parameter.
+        What a line consists of depends on the type of file. For a language data file, each line is a sentence
+        made up of words and punctuation separated by spaces. For a part of speech file, each line
+        is made up of parts of speech separated by spaces, corresponding to the words and punctuation
+        from the language data file.
+
+        Example file structure:
 
-    Example file structure:
+        Language Data File -
+        The dog barked .
+        The cat meowed .
 
-    The/DET dog/NOUN barked/VERB ./PUNC
-    The/DET cat/NOUN meowed/VERB ./PUNC
+        Part of Speech File -
+
+        DET NOUN VERB PUNC
+        DET NOUN VERB PUNC
     """
 
     identifier = "nlp_pos"
     delimiter = Parameter(default="/", type=str)
     include_boundaries = True
+    nlpFileType = Parameter(type=str, help="Must be 'language', 'pos', or 'combined'")
```

Comment:
should I add a `choices` argument to `Parameter`?

Reply:
Sure, I was having trouble figuring out how to implement this since using an Enum didn't really work with the .toml files

Reply:
`Parameter(type=str, choices=['language', 'pos', 'combined'])` after #16 gets merged.